chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -187,7 +187,13 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
                             eventName: BroadcastEvent.StartChatSkippingChatButtonRendering,
                         };
                         BroadcastService.postMessage(chatStartedSkippingChatButtonRendering);
-                        setPreChatAndInitiateChat(chatSDK, props.chatConfig, props.getAuthToken, dispatch, setAdapter);
+                        if (!isUndefinedOrEmpty(state.domainStates?.liveChatContext)
+                            && state.appStates.conversationState === ConversationState.Active) {
+                            const optionalParams = { liveChatContext: state.domainStates?.liveChatContext };
+                            initStartChat(chatSDK, props.chatConfig, props.getAuthToken, dispatch, setAdapter, optionalParams);
+                        } else {
+                            setPreChatAndInitiateChat(chatSDK, props.chatConfig, props.getAuthToken, dispatch, setAdapter, undefined, undefined);
+                        }
```

Comment:
Do we really need to pass undefined here, those are optional params?

Reply:
Removed

Reply:
Removed